index.tpl.html

```diff
@@ -31,6 +31,7 @@ <h1>
 					<div property="description" class="markdown">Description</div>
 					<a href="?tag=[tag]" property="tag" mv-attribute="null" mv-multiple></a>
 					<meta property="repo" mv-edit-placeholder="E.g. [author]/mavo-[id]">
+					<meta mv-if="repo" property="url" mv-edit-placeholder="E.g. https://[author].github.io/[from(repo, '/')]/mavo-[id].js">
```

Comment:
I was thinking that this field will provide the base from which `mavo-[id].js` will be downloaded, i.e. it won't include the filename, so that other files in the repo can use it as a base too. Is there a reason you decided it should include the filename?

Reply:
I was thinking of this field as an alternative way to provide a path to the plugin's main entry point—its JS-file. So I could, for example, provide the transpiled and/or minified version of the plugin to the end-users. For now, the only way I can do that is by transpile and/or minify `mavo-id.js` itself.

And since the `url` field is shown only when the `repo` field is filled in, we can access all the other files through it—all of them are in the provided repo. That's why I didn't think of it as a base URL for all the files in the repo. But I might be mistaken.

Reply:
That is orthogonal. We should support transpiled & minified versions when the author wants to provide them, perhaps we should have fields for their presence too, and they'd be included from the same base URL.

In theory we can use a URL with a filename as a base too, it just doesn't *feel* right. E.g. what if the author specifies a URL that loads a completely different file?